scripts/create_clone

```diff
@@ -24,8 +24,19 @@ def parse_command_line(args):
                         "If not a full pathname, then the case to be cloned"
                         "is assumed to be under then current working directory ")
 
+    parser.add_argument("--user-mods-dir", "-user_mods_dir",
```

Comment:
Should we include here the limitations on user mods if the keepexe flag is used?

Reply:
I agree - I have put in this change and have removed the -user_mods_dir argument and only kept the --user-mods-dir argument. However, this is not consistent with the rest of the arguments in create_clone which all have a -xxx argument that accompanies the --xxx argument.

Reply:
The single dash variations were meant for backward compatibility and so don't need to be included with new options.